src/server/DeviceServer_v2.js

```diff
@@ -181,6 +104,94 @@ class DeviceServer {
     );
   }
 
+  _onNewSocketConnection = async (socket: Socket): Promise<void> => {
+    try {
+      // eslint-disable-next-line no-plusplus
+      const connectionKey = `_${connectionIdCounter++}`;
+      const device = new SparkCore(socket, connectionKey);
+
+      device.on(
+        DEVICE_EVENT_NAMES.READY,
+        (): void => this._onDeviceReady(device),
+      );
+
+      device.on(
+        DEVICE_EVENT_NAMES.DISCONNECT,
+        (): void => this._onDeviceDisconnect(device, connectionKey),
+      );
+
+      device.on(
+        // TODO figure out is this message for subscriptions on public events or
+        // public + private
+        'msg_Subscribe'.toLowerCase(),
```

Comment:
can we just change this to `msg_subscribe` or store these in constants?  I really don't like the `toLowerCase()`